src/base_obs.c

```diff
@@ -174,10 +174,7 @@ static void update_obss(obss_t *new_obss)
   /* Copy over the time. */
   base_obss.tor = new_obss->tor;
 
-  /* Reset the `has_pos` flag. */
```

Comment:
Ah, OK @denniszollo spotted this when we were figuring out the 5Hz stuff.

@denniszollo What is your preference here?